tests/garage/envs/test_normalized_gym.py

```diff
@@ -40,7 +40,9 @@ def test_unflatten(self):
             for _ in range(5):
                 action = self.env.action_space.sample()
                 next_obs, _, done, _ = self.env.step(action)
-                assert self.env.observation_space.flatten(next_obs).shape == \
-                    self.env.observation_space.flat_dim
+                # yapf: disable
```

Comment:
flake8 and yapf don't seem to agree on this. Flake8 puts line break before ==, as recommended by PEP-8, while yapf puts it after.